site/src/components/RequireAuth/RequireAuth.tsx

```diff
@@ -35,16 +35,13 @@ export const RequireAuth: FC = () => {
     return () => {
       axios.interceptors.response.eject(interceptorHandle);
     };
-  }, [authSend]);
+  }, [signOut]);
```

Comment:
One gotcha with the current setup: because `signOut` is defined via a regular function, and it's connected to the context value, this effect is going to run every single time the auth has an update, which probably isn't what we want.

I think wrapping the function in `useCallback` in the AuthContext file might be the quickest fix